### PR TITLE
New translation: Italian

### PR DESCRIPTION
**Pull request by mcaci**
Hello,

If you agree to it I would love to start a new branch of the translation of the documentation for the Italian language. In this PR I have created the folder `site/it` and the first two files `REVIEWERS` where I added myself and the translation of `README.md`.

Would it be ok to you that I start the Italian translation and work on it in the free time?

### CODE REVIEW

**Comment by mcaci** [comment]
@googlebot I signed it!

**Comment by lamberta** [comment]
Nice—you can give it shot 😀
The successful translation projects are able to build a community of contributors and reviewers, and this can take time to build out.
As for docs, I'd start with the quickstart notebook tutorials, then work down the left nav.
Let me know if you have any questions on the docs@tensorflow.org list, or the chat.

**Comment by mcaci** [comment]
Alright, thank you! I'm starting with the quickstart tutorial and the other ones just after

**Comment by mcaci** [comment]
I started adding the beginner tutorial and I will soon add the advanced tutorial :)

**Comment by mcaci** [comment]
Translated the advanced quickstart tutorial. Moving onto other tutorials

**Comment by lamberta** [comment]
Great—Thank you!
Can you create a new pull request for the next tutorial(s)? This will make it easier for potential reviewers and get this merged.

**Comment by mcaci** [comment]
Yes of course, for me we can close this pull request and I put the translations for the next tutorials inside new pull requests.

A good point for having this pr merged, I think, is that potential new contributors will get access to the `it` folder with the translated information on how to contribute and the first examples with the quickstart tutorials.

What do you think?

**Comment by lamberta** [comment]
Oh, yes, sorry for the confusion ...
Please keep this PR open so we can merge it :)
For future notebooks, please create new PRs. I've found that too many notebooks in a single PR is intimidating for reviewers and less likely to get feedback.

**Comment by mcaci** [comment]
No problem, I understand your point. I will create now a separate PR for the first tutorial in the directory `customization\autodiff.ipynb`.

**Comment by mcaci** [comment]
Hi @lamberta , I'm wondering one thing: should I look for other people to have a look and review my translation or will it be done by some of your colleagues who speak Italian?

**Comment by lamberta** [comment]
Hey @mcaci ,
I put out [a call on the docs list](https://groups.google.com/a/tensorflow.org/d/msg/docs/wdDSKDtCA5s/k8xqQ2f1CwAJ) but haven't had any takers, yet. We'll have to build a community outside of any internal TensorFlow folks, so if you know of anyone, please feel free to enlist their help :)
But since building a community can take a while, I can help merge some of the first examples to build momentum. (I just need to check for spam, etc. and can't vouch for the quality of the translation.)
Let's give this one a few more days and then look into getting it in.
Thanks,
Billy

**Comment by mcaci** [comment]
Hey @lamberta thanks for your efforts :) 
I'm thinking to reply in your call by introducing myself and the translation both in Italian and English to keep the conversation up and hopefully attract someone. I could also ask to my wife @maryggiannella who is Italian and a translator to help me out with the check, she was also interested in doing the translation.

Would that be ok?

File: site/it/tutorials/quickstart/beginner.ipynb [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "04QgGZc9bF5D"
+      },
+      "source": [
+        "Questa breve introduzione usa [Keras](https://www.tensorflow.org/guide/keras/overview) per:\n",
+        "\n",
+        "1. Costruire una rete neurale che classifica immagini.\n",
+        "2. Allenare questa rete neurale.\n",
```
Comment (bhack) [inline site/it/tutorials/quickstart/beginner.ipynb]: "Addestrare" probably fit better.

File: site/it/tutorials/quickstart/advanced.ipynb [hunk 2]
```diff
@@ -0,0 +1,418 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "uGih-c2LgbJu"
+      },
+      "source": [
+        "Scegli un metodo di ottimizzazione e una funzione di perdita per l'allenamento:"
```
Comment (bhack) [inline site/it/tutorials/quickstart/advanced.ipynb]: Funzione di perdita doesn't sound so good. Also addestramento IMHO sound Better and It Is more common use.

File: site/it/tutorials/quickstart/beginner.ipynb [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "3wF5wszaj97Y"
+      },
+      "source": [
+        "# Introduzione per chi comincia con TensorFlow 2"
```
Comment (bhack) [inline site/it/tutorials/quickstart/beginner.ipynb]: Guida rapida a Tensorflow 2 per principianti

File: site/it/tutorials/quickstart/advanced.ipynb [hunk 4]
```diff
@@ -0,0 +1,418 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "T4JfEh7kvx6m"
+      },
+      "source": [
+        "Il classificatore di immagini è ora allenato per circa il 98% di precisione su questo insieme di dati. Per impararne di più, leggi i [tutorials di TensorFlow](https://www.tensorflow.org/tutorials)."
```
Comment (bhack) [inline site/it/tutorials/quickstart/advanced.ipynb]: Addestrato al 98% di accuratezza...
Per saperne di più/Per approfondire (sounds better)

File: site/it/tutorials/quickstart/beginner.ipynb [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "ix4mEL65on-w"
+      },
+      "source": [
+        "Allena e valuta il modello:"
```
Comment (bhack) [inline site/it/tutorials/quickstart/beginner.ipynb]: Addestra

File: site/it/tutorials/quickstart/beginner.ipynb [hunk 6]
```diff
@@ -0,0 +1,240 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "BPZ68wASog_I"
+      },
+      "source": [
+        "Costruisci il modello `tf.keras.Sequential` tramite livelli. Scegli un metodo di ottimizzazione e una funzione di perdita per l'allenamento:"
```
Comment (bhack) [inline site/it/tutorials/quickstart/beginner.ipynb]: Generally we don't traslate layer or loss in the day by day use in Italy (in the Deep learning domain)

Comment (lamberta) [reply site/it/tutorials/quickstart/beginner.ipynb]: Thanks for the review.

You might consider adding a "Do not translate" list to the README, similiar to what some other languages are doing (for example, here's the [Korean do not translate list](https://github.com/tensorflow/docs/tree/master/site/ko#korean-translation-guide)). In ML in particular, it seems common to not translate some of these technical terms. But that's a community decision.

Comment (bhack) [reply site/it/tutorials/quickstart/beginner.ipynb]: Yes a do not translate list Is useful.

**Comment by mcaci** [comment]
Thanks for your check @bhack! It's true I chose to over translate to avoid having sentences not translated, but I agree with your suggestions and check for the "Do not translate" list. I'm going to commit them soon.

File: site/it/tutorials/quickstart/advanced.ipynb (hunk 2 above)
Comment (mcaci) [reply site/it/tutorials/quickstart/advanced.ipynb]: I have found the equivalent for the loss function in wikipedia: https://it.wikipedia.org/wiki/Funzione_obiettivo
If you agree will use "Funzione obiettivo" in its place. Meanwhile I change allenamento to addestramento.

File: site/it/tutorials/quickstart/beginner.ipynb (hunk 6 above)
Comment (mcaci) [reply site/it/tutorials/quickstart/beginner.ipynb]: For "loss function" I would ask your feedback on whether "funzione obiettivo" can be acceptable or I leave it as is in English. This also impacts the translation in `advanced.ipynb` where it is present as well.
Meanwhile I added a "Do not translate" section in the readme in Italian.

File: site/it/tutorials/quickstart/advanced.ipynb [hunk 7]
```diff
@@ -0,0 +1,418 @@
...
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "JB6A1vcigsIe"
+      },
+      "source": [
```
Comment (bhack) [inline site/it/tutorials/quickstart/advanced.ipynb]: Precisione or accuratezza? It Is Better to use the same. We have something similar in the same document

File: site/it/README.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+directory. TF 1.x community docs will no longer be updated as we prepare for the
+2.0 release. See
+[the announcement](https://groups.google.com/a/tensorflow.org/d/msg/docs/vO0gQnEXcSM/YK_ybv7tBQAJ).
+
+# Guida alla traduzione italiana
+
+Alcune parole fanno parte di un vocabolario tecnico inglese e pertanto non hanno una traduzione naturale. Per favore *non* traducete le seguenti parole:
+
+*   layer
+*   loss
```
Comment (bhack) [inline site/it/README.md]: Loss Is translated in the document.
Obiettivo Is good enought but loss Is also  very common

File: site/it/REVIEWERS [hunk 9]
```diff
@@ -0,0 +1,8 @@
+# Aggiungi il tuo username GitHub per essere taggato nei commenti su 
+# tutte le pull requests alla cartella site/it. Rimuovi il tuo 
+# username quando desideri.
+#
+# Notifiche per le pull request create sono mandate anche alla lista
+# docs@tensorflow.org: https://groups.google.com/a/tensorflow.org/forum/#!forum/docs
+#
+mcaci
```
Comment (bhack) [inline site/it/REVIEWERS]: You can add also me. 
I can make some double check when I have a slot.

Comment (lamberta) [reply site/it/REVIEWERS]: @bhack Can you make a separate PR to add your name? This way I can verify the CLA and keep folks from adding names without them knowing. Thanks

Comment (bhack) [reply site/it/REVIEWERS]: You can see cla at https://github.com/tensorflow/tensorflow/pull/20437.
But if you want I will add a PR After merge.

File: site/it/README.md (hunk 8 above)
Comment (mcaci) [reply site/it/README.md]: I see, for now I am removing loss from the list of words to not translate. In the future this can change of course

**Comment by mcaci** [comment]
Starting tomorrow, I will be on vacation until next Friday, but I will take any feedbacks on the translation and update the PR when I come back

**Comment by bhack** [comment]
I think It Is enough for a first merge

File: site/it/tutorials/quickstart/advanced.ipynb [hunk 10]
```diff
@@ -0,0 +1,418 @@
...
+        "colab_type": "text",
+        "id": "DUNzJc4jTj6G"
+      },
+      "source": [
+        "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://www.tensorflow.org/tutorials/quickstart/advanced\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />Visualizza su TensorFlow.org</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/tutorials/quickstart/advanced.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Esegui in Google Colab</a>\n",
```
Comment (lamberta) [inline site/it/tutorials/quickstart/advanced.ipynb]: Make sure to update the buttons to point to the 'it' location in GitHub :)
I'll fix this
